dtmi/ted/e1000_ibs01h-1.json

```diff
@@ -0,0 +1,23 @@
+[
```

Comment:
Each file should have one, and only one, interface. Please remove the brackets `[]` from the beginning and end of this file